integration-tests/hibernate-orm-panache/src/test/java/io/quarkus/it/panache/PanacheFunctionalityTest.java

```diff
@@ -76,11 +76,15 @@ public void testJaxbAnnotationTransfer() {
     }
 
     /**
-     * This test is disabled in native mode as there is no interaction with the quarkus integration test endpoint.
+     * _PanacheEntityBase_ has the method _isPersistent_. This method is used by jackson to serialize the attribute *persistent*
+     * in the JSON which is not intended. This test ensures that there something done that this does not happen.
+     * 
+     * This test does not interact with the quarkus application itself. It is just using the jackson ObjectMapper with a
+     * PanacheEntity. Thus this test is enabled in native mode. The test code runs the JVM and not native.
```

Comment:
This line confuses me: `Thus this test is enabled in native mode. The test code runs the JVM and not native.`

Reply:
Ohh... my fault... it should be *disabled* instead of *enabled*...